packages/video_player_avplay/tizen/src/plus_player.cc

```diff
@@ -137,6 +137,13 @@ int64_t PlusPlayer::Create(const std::string &uri,
     is_prebuffer_mode_ = true;
   }
 
+  int64_t start_position = flutter_common::GetValue(
+      create_message.player_options(), "startPosition", 0);
+  if (start_position > 0) {
+    LOG_INFO("[PlusPlayer] Start position: %lld", start_position);
+    Seek(player_, start_position);
```

Comment:
As far as I know, non-seekable content is returned as false, so you can add as the log that the start position didn't work.

Reply:
Done